### ISSUE TITLE
The key.stl file is not printable

### ISSUE THREAD

**Issue by MCMic** [kekstee/3dprint#1]
It seems key.stl is not printable as is…

**Comment by kekstee** [comment]
I have no idea what you tried and why it shouldn't work. 

Also I am not capable of testing this myself at the moment.

**Comment by myoung34** [comment]
I agree, the key is pretty impossible to print. The base comes out mangled all the time. I'll add the STL i refactored.
